<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Resumen</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_titulo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resumen de la Ronda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ptje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puntaje obtenido:                               </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ptje_acumulado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puntaje acumulado:                           </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxcombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Máximo combo:                                   </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fallados">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cantidad de pasos fallados:           </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_porcentajeaprob">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Porcentaje de aprobación:              </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(134, 187, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dato1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 pts</string>
   </property>
  </widget>
  <widget class="QLabel" name="dato2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 pts</string>
   </property>
  </widget>
  <widget class="QLabel" name="dato3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 pts</string>
   </property>
  </widget>
  <widget class="QLabel" name="dato4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 pts</string>
   </property>
  </widget>
  <widget class="QLabel" name="dato5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 pts</string>
   </property>
  </widget>
  <zorder>label_fondo</zorder>
  <zorder>label_titulo</zorder>
  <zorder>boton_volver</zorder>
  <zorder>label_ptje</zorder>
  <zorder>label_ptje_acumulado</zorder>
  <zorder>label_maxcombo</zorder>
  <zorder>label_fallados</zorder>
  <zorder>label_porcentajeaprob</zorder>
  <zorder>dato1</zorder>
  <zorder>dato2</zorder>
  <zorder>dato3</zorder>
  <zorder>dato4</zorder>
  <zorder>dato5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
